vue/templates/browserslistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
```

Comment:
Should this be moved into the `_package.json` like the other generators?
(I know this this Vue template code is from another contributor's older PR)